sdk/storage/azure_storage_blob/src/lib.rs

```diff
@@ -3,13 +3,15 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 // Code generated by Microsoft (R) Rust Code Generator. DO NOT EDIT.
 
+pub mod clients;
 mod generated;
+pub mod pipeline;
 
-pub use crate::generated::clients::*;
+pub mod models;
 
-pub mod models {
-    pub use crate::generated::enums::*;
-    pub use crate::generated::models::*;
-}
+pub use crate::generated::clients::*;
+pub use blob_client::BlobClient as GeneratedBlobClient;
 
-pub use blob_client::BlobClient;
+pub use crate::blob_client::BlobClientOptions;
+pub use crate::clients::BlobClient;
```

Comment:
Why are these even defined in a separate module? Don't do 1 type per file like .NET. That's not idiomatic and makes things harder since every file is a separate module which then requires changing accessibility. The only types that should be public are your clients, client options, method options, and request models in the root, and remaining models (response models, basically) in a `models` public module.

Reply:
L15 here is the generated model. Is there a better way to bring `BlobClientOptions` up to the root so that it can be imported as `use azure_storage_blob::BlobClientOptions`

L16 is our `BlobClient`, so I can take the feedback there. That'll be exported for free after taking an above feedback item you've suggested: `pub use clients::*;`